lib/tasks/update_friend_volunteer_associations.rake

```diff
@@ -0,0 +1,5 @@
+desc 'Delete friend-volunteer associations that are >= 3 months old'
+task :update_friend_volunteer_associations => :environment do
+  old_relations = UserFriendAssociation.where('created_at < ?', 3.month.from_now)
+  old.relations.delete
```

Comment:
It looks like, this was meant to be old_relations, rather than old.relations.  I tend to use .destroy for things like this because .destroy also calls the callbacks.  In this case, there are no callbacks to worry about, so either .destroy or .delete will work for our purposes.

Reply:
I'll change it to `destroy` to keep it consistent with the rest of the codebase.